### PR TITLE
Documentation for the Timeline data visualization

### PR DESCRIPTION
**Pull request by serenamarie125**

### CODE REVIEW

File: pattern_library/data-visualization/timeline/timeline-design.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Timeline
+
+## Timeline
+![Image of standard timeline](timeline-callout.png)
+
+1. **Horizontal Axis Labels**: The horizontal axis labels display time
+1. **Vertical Axis Labels**: The vertical axes labels represent the event type being displayed in that row.  The label can optionally include the total number of event occurrences in the specified timeline.
+1. **Hover Indicator**: When hovering over the timeline, a hover indicator is displayed to indicate the date and time of that time interval.
```
Comment (LHinson) [inline pattern_library/data-visualization/timeline/timeline-design.md]: This item seems like a child of the "Interaction" call out

File: pattern_library/data-visualization/timeline/timeline-design.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+1. **Events**
+  * **Interaction** (optional):
+    * If supported, clicking on an event occurrence will display details of the event.
```
Comment (LHinson) [inline pattern_library/data-visualization/timeline/timeline-design.md]: "indicator" as specified/defined later in the documentation vs "occurrences"

File: pattern_library/data-visualization/timeline/timeline-design.md (hunk 1 above)
Comment (LHinson) [inline pattern_library/data-visualization/timeline/timeline-design.md]: Use "indicator" as specified/defined later in the documentation vs "occurrences". Also remove "of that time interval" as I don't think it's necessary.

Comment (LHinson) [inline pattern_library/data-visualization/timeline/timeline-design.md]: "A gray vertical line" vs "hover indicator" is more specific.

File: pattern_library/data-visualization/timeline/timeline-design.md (hunk 2 above)
Comment (serenamarie125) [inline pattern_library/data-visualization/timeline/timeline-design.md]: Noted @LHinson I will replace all references to "occurrence" with "indicator" where applicable.

File: pattern_library/data-visualization/timeline/timeline-design.md (hunk 1 above)
Comment (serenamarie125) [inline pattern_library/data-visualization/timeline/timeline-design.md]: @LHinson Fixed & updated PR

File: pattern_library/data-visualization/timeline/timeline-design.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+1. **Vertical Axis Labels**: The vertical axes labels represent the event type being displayed in that row.  The label can optionally include the total number of event indicators in the specified timeline.
+1. **Hover Indicator**: When hovering over the timeline, a gray vertical line is displayed with a tooltip indicating the date and time of that point on the timeline.
+1. **Events**
+  * **Interaction** (optional):
+    * If supported, clicking on an event indicator will display details of the event.  
+    * If supported, right clicking on an event indicator will bring up a contextual menu with associated actions.
+  * **Event Indicators**: Events can be visually represented as shapes or icons.  A user can view information related to a specific event by hovering over it.  To help the user see which point they are hovering, the dot expands. Additionally, a tooltip should appear with the associated values for that specific event.
+  * **Color**: For recommendations on colors, see the [color palette](https://www.patternfly.org/styles/color-palette/).
+1. **Interactive Chart**
+  * Click and drag the chart left or right to navigate back or forward in time.
```
Comment (LHinson) [inline pattern_library/data-visualization/timeline/timeline-design.md]: *backwards or forwards in time

File: pattern_library/data-visualization/timeline/timeline-design.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  * Use the mouse wheel to zoom in and out of the time range.
+  * Note: The view context in the navigation viewport automatically adjusts its size and position to match the data shown.
+1. **Zoom Slider**
+  * **+** : Incrementally zooms in to the time range
```
Comment (LHinson) [inline pattern_library/data-visualization/timeline/timeline-design.md]: *into vs in to

File: pattern_library/data-visualization/timeline/timeline-design.md (hunk 3 above)
Comment (serenamarie125) [inline pattern_library/data-visualization/timeline/timeline-design.md]: @LHinson Fixed & updated PR

File: pattern_library/data-visualization/timeline/timeline-design.md (hunk 4 above)
Comment (serenamarie125) [inline pattern_library/data-visualization/timeline/timeline-design.md]: @LHinson Fixed & updated PR

File: pattern_library/data-visualization/timeline/timeline-design.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  * Drag it left / right to scroll through the data on the main chart.
+  * Click and drag the left and right edges to increase / decrease the amount of data shown.
+  * Click off of the view context in the viewport to show the full data set in the main chart.
+  * Click and drag on the viewport to create a new view context.
+
+
+## Timeline Hover
+![Image of timeline hover](timeline-hover.png)
+
+1. **Tooltip**: We recommend that the event type and object name associated with the event and date and time the event occurred are displayed on hover.
```
Comment (LHinson) [inline pattern_library/data-visualization/timeline/timeline-design.md]: This sentence is a little confusing / not clear.

File: pattern_library/data-visualization/timeline/timeline-overview.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Timeline
+
+## Overview
+
+A timeline is a visualization of events over time.  Event types are shown as individual rows on the timeline.  Each occurrence of an event is displayed on the associated row by a shape/icon and color.
+
+The most common use case for a timeline is:
+* Users need to view a timeline of events associated with a specific object.
```
Comment (LHinson) [inline pattern_library/data-visualization/timeline/timeline-overview.md]: Since there is only one uses case, no need to bullet it.

**Comment by LHinson** [comment]
I've made my comments - just small things here and there unrelated to the design itself. Thanks @serenamarie125! 

@andresgalante please review.

**Comment by serenamarie125** [comment]
Changed to "Tooltips should include the event type, name of the object as well as the date and time the event occurred."

File: pattern_library/data-visualization/timeline/timeline-overview.md (hunk 6 above)
Comment (serenamarie125) [inline pattern_library/data-visualization/timeline/timeline-overview.md]: All set @LHinson, updated the PR

File: pattern_library/data-visualization/timeline/timeline-design.md (hunk 5 above)
Comment (serenamarie125) [inline pattern_library/data-visualization/timeline/timeline-design.md]: @LHinson, changed to "Tooltips should include the event type, name of the object as well as the date and time the event occurred."

File: pattern_library/data-visualization/timeline/timeline-design.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
+# Timeline
+
+## Timeline
```
Comment (andresgalante) [inline pattern_library/data-visualization/timeline/timeline-design.md]: Do we need to repeat the title as a subtitle here?

**Comment by LHinson** [comment]
Agreed @andresgalante. Actually we should make sure this now aligns with the template that Chris just completed https://github.com/patternfly/patternfly-design/pull/12/files

**Comment by LHinson** [comment]
@cshinn can you help out here and update the template so we can merge?

**Pull request merged**